src/features/vote/ActiveRequestsForm.tsx

```diff
@@ -1,92 +1,650 @@
 /** @jsxImportSource @emotion/react */
-import { FC } from "react";
+import { FC, useCallback, useContext, useState, useEffect } from "react";
```

Comment:
i knwo this file is large, but i dont see anything particularly wrong with it. maybe there is some functionality which could be pulled out, but i dont have any suggestions at the moment

Reply:
We could break out the modal into its own file, which might help actually. But yeah, it's also a complex component regardless, as it's the form and the view all packed into one.

Reply:
yea whatever you think is best, anything would help here